andela/might-guy-cfh
# #141409269 Re-design gameplay screen

#### What does this PR do?
- Merges changes made to `staging` branch
- Finishes User Story #141409269

#### Description of Task to be completed?
- Background picture for gameplay changed
- Interface revealed when users opt to start a new game
- Merge conflict resolved.
- In-app notification added to game play screen

#### How should this be manually tested?
- Users should visit [review app](https://might-guy-cfh-staging-pr-44.herokuapp.com/#!/)
- Click `Play as Guest`
- New game play interface would be revealed.

#### Any background context you want to provide?
- Old game play screen is dated with objects not properly aligned
- Different font family implemented

#### What are the relevant pivotal tracker stories?
- #141409269
[Fixes #141409269]
